lacework/resource_lacework_vulnerability_exception_host.go

```diff
@@ -218,17 +218,19 @@ func resourceLaceworkVulnerabilityExceptionHostCreate(d *schema.ResourceData, me
 	var (
 		lacework   = meta.(*api.Client)
 		severities = castAttributeToStringSlice(d, "vulnerability_criteria.0.severities")
+		fixable    = d.Get("vulnerability_criteria.0.fixable").(bool)
```

Comment:
this looks different from container. can we fix host and do what you did in container?